bin/react-to-product-owners-yml-changes.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")"
+cd ..
+gh repo clone https://github.com/getsentry/security-as-code /tmp/security-as-code
```

Comment:
Do we need to `cd` into the right repo after making the tmp file? Seems like this works without it, curious as to why

Reply:
nope, we don't care about anything in the `security-as-code` repo except the `product-owners.yml` file, which is passed in as an arg for the `react-to-product-owners-yml-changes.py` script.